src/ui_basic/scrollbar.cc

```diff
@@ -409,13 +410,16 @@ void Scrollbar::think() {
 	}
 }
 
-bool Scrollbar::handle_mousewheel(uint32_t, int32_t, int32_t y) {
-	if (y < 0) {
-		action(Area::Plus);
-	} else {
-		action(Area::Minus);
+bool Scrollbar::handle_mousewheel(int32_t, int32_t y, uint16_t modstate) {
+	if (matches_keymod(modstate, KMOD_NONE) && y != 0) {
+		if (y < 0) {
```

Comment:
Does this also need `get_mousewheel_change()`?

Reply:
Yes, I was considering it, and it should probably use it too.

I kept it as it was for now because I intended to revisit this, because I'm not sure what to do with horizontal scrolling. As far as I can tell, only vertical scrollbars are used currently, but the widget is ready for horizontal too. But yes, `get_mousewheel_change()` can be used in any case, only it will need to be configurable if there will be horizontal scrollbars.